raiden/tests/integration/network/transport/test_matrix_transport_assumptions.py

```diff
@@ -259,6 +264,72 @@ def test_assumption_matrix_returns_same_id_for_same_filter_payload(chain_id, loc
     assert first_sync_filter_id == second_sync_filter_id
 
 
+@pytest.mark.parametrize("number_of_transports", [3])
+@pytest.mark.parametrize("matrix_server_count", [1])
+def test_admin_is_allowed_to_kick(matrix_transports, local_matrix_servers):
+    server_name = local_matrix_servers[0].netloc
+    admin_credentials = get_admin_credentials(server_name)
+    broadcast_room_name = "raiden_337_discovery"
```

Comment:
Slight nitpick: `337` is defined in `raiden.tests.utils.factories.UNIT_CHAIN_ID` and `raiden.network.transport.matrix.utils.make_room_alias()` can be used build the room alias